openpype/modules/archiver/archiver_module.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from openpype.modules import OpenPypeModule, ITrayService
+
+
+class ArchiverModule(OpenPypeModule, ITrayService):
```

Comment:
`ITrayService` seems useless

Reply:
Hadn't seen that comment, answered and solved [here](https://github.com/quadproduction/OpenPype/pull/752#issuecomment-2135116970)